### DISCUSSION TITLE
Scope layout and blocks?

### DISCUSSION THREAD

**Discussion by luckykenlin** [Z3d0X/filament-fabricator#110] [General]
Thank you for the fantastic plugin!

Currently, I find myself in a situation where I've successfully constructed a basic landing page using your plugin. 
Now, I'm eager to create an additional template that possesses distinct layouts and page blocks. 
In pursuit of this goal, I'm seeking guidance on the scoping possibilities within your plugin.

I am interested in understanding the feasibility of enabling layouts and blocks to dynamically adapt and change based on the selected template.

Specifically, I'm curious to know if it's feasible to scope the following elements:

- Layouts
- PageBlocks
- Meta data
- Script
- Stylesheets

Any guidance or pointers toward relevant resources would be immensely appreciated.

Thank you once again for your remarkable work and for your attention to this query.

**Comment by KevenOuellet** [comment]
Check the pro tip here about conditionally showing blocks:  https://filamentphp.com/plugins/z3d0x-fabricator#page-block-schema
Does it help ?
